libs/components/lookup/src/lib/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -108,7 +109,7 @@ export class SkyAutocompleteInputDirective
   }
 
   public get textChanges(): Observable<SkyAutocompleteInputTextChange> {
-    return this.#textChangesObs;
+    return this.#textChanges.asObservable();
```

Comment:
These were set to `this.#textChangesObs;` in the constructor so that the value is created once, and not every time `get textChanges` is called.

Reply:
Ahh ok, reverted.